copylot/hardware/orca_camera/camera.py

```diff
@@ -16,8 +51,21 @@ class OrcaCamera:
     """
 
     def __init__(self, camera_index: int = 0):
-
+        self.trigger_source = None
+        self.trigger_polarity = None
         self._camera_index = camera_index
+        self.dcam = None
+        self.exposure_time_ms = None
```

Comment:
a lot of these member attributes like the one for exposure will be more suitable to implement as a Python property. Please read more about Python properties here: https://docs.python.org/3/library/functions.html#property , let's do this in a separate PR together @xiyuyi !